en/getting-started.md

```diff
@@ -1,10 +1,104 @@
+- Indigo-Components - the library containing a vast set of Components to layout your intricate design ideas and eventaully, after a few design iterations, generate code assets from. Through the overrides section you can choose from numerous presets compatible with [Ignite UI for Angular](https://www.infragistics.com/products/ignite-ui-angular) and our code generation engine.
+- Indigo-Patterns - the library where components shape up meaningful patterns for productive application design. All you need to do, is cherry pick a few layouts and focus on designing a logical user journey and customizing the design to make it more appealing for your audience.
+
+> [!INFO]
+> If you are part of a larger organization and collaborate with other designers or need to follow a certain compnay-wide guideline, we would advise you to put the libraries on a shared folder in the cloud, but one that also exists on your local drive and is synced via the could provided application. This will mean that any changes made to the libraries e.g. when you implement your custom company branding that needs to be used consistently across all your digital products, will propagate to everyone, who has access to this folder and has been using the libraries to create amazing experiences.
+
+In order to add the Libraries in Sketch follow these
+
+1.  Open up the Sketch app on your Mac, click on the `Sketch` item from the Apple Menu and select `Preferences`.
+2.  In the dialog that appears, select the `Libraries` tab and press the `Add Library...` button at the bottom.
+3.  Another dialog will show up, allowing you to navigate to the folder on your local drive, where you have placed the Libraries.
+4.  Select all three of them and click `Open`, which will close this second dialog window.
+5.  Now that they all show up in the list in the Preferences dialog, make sure that they all are checked and close it .
+6.  If the Libraries have been added successfully, they will all show up in the `Symbols` section of the `Insert` menu in Sketch.
+
+### Creating your First Indigo Design Artboard
+
+Now that you have added the Indigo Design Libraries, you are all set to start crafting amazing experiences. Our first design will libraries will be for a simple registration screen with a Navbar, and a beautiful image for background on top of which we will design a simple form consisting of a little bit of Text, a few Inputs and a Button to submit the user details. So let's find out what a productivity boost the Indigo Design system brings to the table.
+
+1.  Open up the Sketch app on your Mac and create a new file. Press the `A` key and from the right panel select an artboard size from the panel on the right. In this example we will be using an `iPhone 8` in portrait orientation, which will give us a 375 by 667 artboard to design for. If you don't see this preset arboard, choose `Apple Devices` from the dropdown at the top of the panel.
```

Comment:
If you don't see this preset artboard, choose `Apple Devices` from the dropdown at the top of the panel.